micro-ui/web/micro-ui-internals/packages/modules/pgr/src/pages/employee/ComplaintDetails.js

```
@@ -0,0 +1,505 @@
+    setTimeout(() => setToast(false), 10000);
+  }
+
+  function closeToast() {
+    setToast(false);
+  }
+
+  if (isLoading || workflowDetails.isLoading || loader) {
+    return <Loader />;
+  }
+
+  if (workflowDetails.isError) return <React.Fragment>{workflowDetails.error}</React.Fragment>;
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)